scrapers/FetishNetwork.yml

```diff
@@ -0,0 +1,64 @@
+name: Fetishnetwork
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fetishnetwork.com/t2/showgal
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      # Title: //div[contains(@class,"detail-area")]//h3
+      Title:
+        selector: //title
+        postProcess:
+          - replace:
+              - regex: ^(.+)(?:\s\|).+$
+                with: $1
+      Details:
+        selector: //div[contains(@class,"description-long-text")]//text()
+        postProcess:
+          - replace:
+              - regex: Description:\s?
+                with:
+              - regex: �
```

Comment:
Do you by any chance have the url for this scene handy?
I am curious to what `//div[contains(@class,"description-long-text")]` would show

Reply:
Some older scenes have them, e.g.:
`http://www.fetishnetwork.com/t2/showgal.php?g=content/flash/p0007_s0018/p0007_s0018_0058breastfest_videos/29269/506_1&s=1100&uvar=typein.4.10.10.0.0.0.0.0`